src/ceph-disk/ceph_disk/main.py

```diff
@@ -1536,9 +1546,7 @@ def zap(dev):
     """
     Destroy the partition table and content of a given disk.
     """
-    dev = os.path.realpath(dev)
```

Comment:
@dachary 
Oke, I'll go over the PR with a fine comb, because I think I also killed af few `stat()` versus `lstat()`, and that might then also matter. I'll put back the leader part of the function in the code.
I thought that it was also a nice cleanup as extra bonus, but obviously not.